pkg/kanctl/actionset.go

```diff
@@ -342,6 +355,25 @@ func parseProfile(cmd *cobra.Command, ns string) (*crv1alpha1.ObjectReference, e
 	}, nil
 }
 
+func parseRepositoryServer(cmd *cobra.Command, ns string) (*crv1alpha1.ObjectReference, error) {
```

Comment:
We don't  have to do anything here, but I would rather prefer `objectRefFromRepoServer` instead of `parseRepositoryServer`.
But I understand is ok to keep it as it is for consistency. Maybe just add a comment that it returns object reference for passed repo server.

Reply:
Added the comment !